OpenAPIService/OpenApiService.cs

```diff
@@ -774,6 +774,15 @@ private static void CopyReferences(OpenApiDocument target)
         private static bool AddReferences(OpenApiComponents newComponents, OpenApiComponents target)
         {
             var moreStuff = false;
+            moreStuff = AddComponentsSchemas(newComponents, target, moreStuff);
+            moreStuff = AddComponentsParameters(newComponents, target, moreStuff);
+            moreStuff = AddComponentsResponses(newComponents, target, moreStuff);
+            moreStuff = AddComponentsRequestBodies(newComponents, target, moreStuff);
```

Comment:
```suggestion
            moreStuff |= AddComponentsSchemas(newComponents, target, moreStuff);
            moreStuff |= AddComponentsParameters(newComponents, target, moreStuff);
            moreStuff |= AddComponentsResponses(newComponents, target, moreStuff);
            moreStuff |= AddComponentsRequestBodies(newComponents, target, moreStuff);
```

(I'm not sure resetting the more stuff switch after each method was what was desired)